tests/util/mod.rs

```diff
@@ -71,21 +71,20 @@ struct CaseTraceLogger {
 impl Drain for CaseTraceLogger {
     type Ok = ();
     type Err = slog::Never;
-    fn log(&self, record: &Record, values: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
-        let tag = util::get_tag_from_thread_name().unwrap_or_else(|| "".into());
+    fn log(&self, record: &Record, _: &OwnedKVList) -> Result<Self::Ok, Self::Err> {
```

Comment:
Should we consider the values(key-value lists of a `Logger` it was executed on)?

Reply:
Seems we didn't use this feature. For now `values` always be `()`.